==== 测试用例 9 (failed) ====

输入:
328
3 0 1 -3 1 -3 -1 -3 3 1 -2 2 1 -2 -2 1 -4 1 0 1 -1 -4 -1 0 0 -3 2 -4 2 -1 -3 0 2 0 -3 -3 1 1 1 -3 -4 -4 3 1 2 1 2 3 -4 -2 -3 -1 -1 -2 1 2 -4 -2 -1 -3 1 -3 -4 3 -3 -2 -2 -2 3 -1 -4 3 -4 3 2 0 2 3 -1 -4 3 0 -2 -3 -2 1 3 -4 2 -3 2 -1 -1 1 2 -3 0 2 -4 0 -4 3 -4 2 -1 0 2 -4 -1 -3 3 3 2 0 -2 3 1 -4 1 -3 1 -2 3 -4 0 1 3 2 1 -4 0 3 2 3 -3 0 -3 -3 -4 -1 -2 1 2 -2 -1 3 -4 -1 -1 -4 3 -1 -2 -1 -1 -4 -4 2 -3 0 -4 -1 1 1 -2 -4 -1 2 -3 0 -3 -1 2 -3 2 2 0 1 -1 -3 -3 -4 -2 2 -1 -2 2 -2 -3 0 -2 -3 3 3 -3 -2 -4 -1 1 -4 1 -3 2 -3 2 -4 2 0 2 -3 1 0 1 0 -4 2 2 3 2 -4 -2 -158 -3 -2 -2 -3 -2 3 3 0 0 0 0 -2 -4 -2 2 2 0 -2 -3 3 -1 1 0 3 3 -2 1 -2 2 2 -1 -4 2 1 -1 0 -4 1 3 -3 3 -1 3 1 -1 3 0 -1 3 0 0 -4 3 0 0 0 -3 0 -3 3 0 -3 0 2 2 -3 -3 3 -2 -473 1 -3 -3 -1 0 1 1 -2 -3 0 0 -3 2 -3 -1 -4 -1 -2 2 2 1 -2 -1 -4 2 3 -2 0 2 -3 1 2 -3 1 -3 3
期望输出:
-473
实际输出:
ERROR: Process exited with code 101: thread 'main' panicked at /home/xiaofan/dev_25/transproj/TranslateToRust/llm_translated/grok-2-1212-default/weekly_contest_426_p2_cpp.rs:40:50:
called `Result::unwrap()` on an `Err` value: ParseIntError { kind: InvalidDigit }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

运行时间: 0.00 ms
